core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/TargetedSequencing.java

```diff
@@ -2,40 +2,64 @@
 
 import java.util.Date;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
 import com.eaglegenomics.simlims.core.User;
 
 import uk.ac.bbsrc.tgac.miso.core.data.impl.kit.KitDescriptor;
 
+@Entity
+@Table(name = "TargetedSequencing")
 public class TargetedSequencing {
 
   public static final Long UNSAVED_ID = 0L;
 
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
   private Long targetedSequencingId;
 
   /** Name displayed in UI and associated with targeted sequencing bed file. */
```

Comment:
Out of scope, but turning these comments into Javadocs on the getters and setters would be nice :grin: